xlb/operator/boundary_condition/bc_regularized.py

```diff
@@ -62,6 +62,8 @@ def __init__(
             mesh_vertices,
         )
 
+        self.id = boundary_condition_registry.register_boundary_condition(__class__.__name__)
```

Comment:
Please remove this and change the similar line in bc_zouhe.py to the following:
`self.id = boundary_condition_registry.register_boundary_condition(self.__class__.__name__ + "_" + bc_type)`
adding `self` does the trick for the child class.

Reply:
I don't think this is a good solution it's more confusing and only works for Zouhe than what we have right now which is at least explicit. There should be a better solution otherwise I prefer what we have right now (at least you don't have to dig into Zouhe to find it).

Reply:
We have this line for all BCs. Let's keep it this way it is consistent as Regularized is a new BC like others.

Reply:
Found a unified way to apply this to avoid the issue. I'll push it.